### PR TITLE
mobile extend better bigger small margins

### PR DESCRIPTION
**Pull request by tenisakb**

### CODE REVIEW

File: src/components/download-form/download-form-slide.tsx [hunk 1]
```diff
@@ -42,6 +45,12 @@ const HandleTextInput = styled.input`
     outline: none !important;
     box-shadow: none !important;
   }
+
+  @media only screen and (max-width: 776px) {
+    width: 100%;
+    padding-left: 0px !important;
```
Comment (EdmundMai) [inline src/components/download-form/download-form-slide.tsx]: find a way to do this without `!important`

File: src/components/download-form/download-form-slide.tsx [hunk 2]
```diff
@@ -42,6 +45,12 @@ const HandleTextInput = styled.input`
     outline: none !important;
     box-shadow: none !important;
   }
+
+  @media only screen and (max-width: 776px) {
```
Comment (EdmundMai) [inline src/components/download-form/download-form-slide.tsx]: let's extract `776` into a config var and point to it across the whole app. so something like:

`@media only screen and (max-width: ${CONFIG.DESKTOP_WIDTH}px) {`

Comment (tenisakb) [reply src/components/download-form/download-form-slide.tsx]: solved

File: src/components/download-form/download-form-slide.tsx (hunk 1 above)
Comment (tenisakb) [reply src/components/download-form/download-form-slide.tsx]: solved

File: src/components/download-form/download-form-slide.tsx (hunk 2 above)
Comment (EdmundMai) [reply src/components/download-form/download-form-slide.tsx]: nice looks good

**Pull request closed without merging**